### PR TITLE
1100-Enhancement-DatePicker-supports-nullable-(Selected)-Date

### PR DESCRIPTION
**Pull request by ghost**
### Description of Change ###

Implements #

### Additions made ###

* Adds 

### PR Checklist ###

- [ ] Targets the correct branch 
- [ ] Tests are passing (or failures are unrelated)
- [ ] Targets a single property for a single control (or intertwined few properties)
- [ ] Adds the property to the appropriate interface
- [ ] Avoids any changes not essential to the handler property
- [ ] Adds the mapping to the PropertyMapper in the handler
- [ ] Adds the mapping method to the Android, iOS, and Standard aspects of the handler
- [ ] Implements the actual property updates (usually in extension methods in the Platform section of Core)
- [ ] Tags ported renderer methods with [PortHandler]
- [ ] Adds an example of the property to the sample project (MainPage)
- [ ] Adds the property to the stub class
- [ ] Implements basic property tests in DeviceTests

#### Does this PR touch anything that might affect accessibility?
- [ ] Does this PR introduce a new control? (If yes, add an example using SemanticProperties to the SemanticsPage)
- [ ] APIs that modify focusability?
- [ ] APIs that modify any text property on a control?
- [ ] Does this PR modify view nesting or view arrangement in anyway?
- [ ] Is there the smallest possibility that your PR will change accessibility? 
- [ ] I'm not sure, please help me

If any of the above checkboxes apply to your PR, then the PR will need to provide testing to demonstrate that accessibility still works.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/maui?pullRequest=9727) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/maui?pullRequest=9727) before we can accept your contribution.<br/><br/>:x: ryanwmitchell [sign now](https://cla.dotnetfoundation.org/dotnet/maui?pullRequest=9727)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/maui?pullRequest=9727) it.</sub>

File: src/Compatibility/Core/src/Windows/DatePickerRenderer.cs [hunk 1]
```diff
@@ -126,7 +126,12 @@ void OnControlDateChanged(object sender, DatePickerValueChangedEventArgs e)
 			if (Element == null)
 				return;
 
-			if (Element.Date.CompareTo(e.NewDate.Date) != 0)
+			var elementDate = Element.Date;
```
Comment (duindain) [inline src/Compatibility/Core/src/Windows/DatePickerRenderer.cs:129]: var elementDate = Element?.Date

if(elementDate == null)
    return

**Comment by jfversluis** [comment]
Hey @ryanwmitchell thank you so much for doing this and sorry we didn't get to this earlier. I'm on this now! 

Are you still interested in taking this home? If so, let's work on it! If not, also fine, then I will take over and see if we can get it in a mergeable state. Thanks!

**Comment by jfversluis** [comment]
No response and PR grew too stale to be recoverable at this point. Let's close it and revisit later. Thanks for the time and effort here!

**Comment by eder-careplus** [comment]
@jfversluis this is a much need thing on MAUI. A Nullable DatePicker is a must have for a lot of scenarios. Please raise a priority on it.

**Pull request closed without merging**